daemon/images/images.go

```diff
@@ -317,14 +317,14 @@ func (i *ImageService) SquashImage(id, parent string) (string, error) {
 		newImage.History[i] = hi
 	}
 
-	now := time.Now()
 	var historyComment string
 	if len(parent) > 0 {
 		historyComment = fmt.Sprintf("merge %s to %s", id, parent)
 	} else {
 		historyComment = fmt.Sprintf("create new from %s", id)
 	}
 
+	now := time.Now().UTC()
```

Comment:
Discussing with @tonistiigi, and he's wondering if this actually makes a difference in the image config/history.

I'll check if it makes a difference in the result